.github/workflows/main.yml

```diff
@@ -187,8 +187,109 @@ jobs:
         target: debug
         tags: ${{ steps.meta.outputs.tags }}
 
+  copy_offload_production:
+    needs: [production]
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: "Build context"
+      run: |
+        echo "ref is ${{ github.ref }}"
+        echo "ref_type is ${{ github.ref_type }}"
+
+    - name: "Checkout repository"
+      id: checkout_repo
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: "Lowercase repository name for docker build"
+      id: lowercase-repository-name
+      run: echo "REPO_NAME=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
+
+    - name: "Set tags for main/master"
+      id: set_tags
+      run: |
+        echo "VERSION_TAG=$(./git-version-gen | grep -v UNKNOWN)" >> ${GITHUB_ENV}
+        echo "TEST_TAG=$(git rev-parse HEAD)-test" >> ${GITHUB_ENV}
+        echo "SHA_TAG=$(git rev-parse HEAD)" >> ${GITHUB_ENV}
+        echo "${GITHUB_ENV}:"
+        cat ${GITHUB_ENV}
+      shell: bash
+
+    - name: "Verify auto-generated files"
+      run: |
+          make manifests generate
+          git status
```

Comment:
Why do you run `git status` twice here?

Reply:
When it fails, in the next line, I find that I usually want to know what the git-status output looked like. So I put it in the output of the job.